py_ecc/fields/field_elements.py

```diff
@@ -254,8 +270,10 @@ def __rmul__(self: T_FQP, other: Union[int, T_FQP]) -> T_FQP:
         return self * other
 
     def __div__(self: T_FQP, other: Union[int, T_FQP]) -> T_FQP:
-        if isinstance(other, int_types_or_FQ):
-            return type(self)([c / other for c in self.coeffs])
+        if isinstance(other, int):
+            return type(self)(
+                [c / other if isinstance(c, FQ) else c // other for c in self.coeffs]
```

Comment:
The return type requires that the list be `int`s. `/` is defined for `FQ` such that it will be an int, but in the case that `c` is an int, `//` is needed to guarantee the `int`. The other option I thought would be to make the `int` an `FQ` and then just use `/`.